yt_dlp/extractor/cbc.py

```diff
@@ -46,19 +35,6 @@ class CBCIE(InfoExtractor):
             'uploader': 'CBCC-NEW',
         },
         'skip': 'Geo-restricted to Canada',
-    }, {
-        # with clipId, feed available via tpfeed.cbc.ca and feed.theplatform.com
-        'url': 'http://www.cbc.ca/22minutes/videos/22-minutes-update/22-minutes-update-episode-4',
-        'md5': '162adfa070274b144f4fdc3c3b8207db',
-        'info_dict': {
-            'id': '2414435309',
-            'ext': 'mp4',
-            'title': '22 Minutes Update: What Not To Wear Quebec',
-            'description': "This week's latest Canadian top political story is What Not To Wear Quebec.",
-            'upload_date': '20131025',
-            'uploader': 'CBCC-NEW',
-            'timestamp': 1382717907,
-        },
```

Comment:
If removing this because the link isnt working, add a new test for `with clipId, feed available via tpfeed.cbc.ca and feed.theplatform.com`

Reply:
I haven't been able to find a video that satisfies these requirements.

Reply:
In that case, leave the test there with something like `'skip': 'No longer available'`. This will make sure future devs know that this is a case to account for